homework/shelkovich_ilya/lookUpArrayService/src/main/java/com/tutrit/java/quickstart/service/LookUpArrayService.java

```diff
@@ -0,0 +1,72 @@
+package com.tutrit.java.quickstart.service;
+
+public class LookUpArrayService {
+
+
+    public int findMax(int[] array) {
+        if (array == null || array.length == 0) {
+            return 0;
+        }
+        int max = array[0];
+        for (var i = 1; i < array.length; i++) {
+            if (array[i] > max) {
+                max = array[i];
+            }
+        }
```

Comment:
this is a service logic. It could be divided from previous .

Reply:
Got it! Thx